zkevm-circuits/src/pi_circuit.rs

```diff
@@ -226,19 +226,19 @@ impl<F: Field> SubCircuitConfig<F> for PiCircuitConfig<F> {
         let tx_value = tx_table.value;
         let tag = tx_table.tag;
         let index = tx_table.index;
-        let tx_id_inv = meta.advice_column();
-        let tx_value_inv = meta.advice_column();
-        let tx_id_diff_inv = meta.advice_column();
+        let tx_id_inv = meta.advice_column_in(SecondPhase);
+        let tx_value_inv = meta.advice_column_in(SecondPhase);
```

Comment:
Ditto. See https://github.com/privacy-scaling-explorations/zkevm-circuits/blob/5ab275565ddd35e7192623f131d0737784f04467/zkevm-circuits/src/pi_circuit.rs#L630-L631

Reply:
Ohh I see! The `tx_value` variable is holding the RLC already!! That makes sense! Ignore this then.